Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -1303,7 +1312,7 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
 
   override func visit(_ node: AvailabilityLabeledArgumentSyntax) -> SyntaxVisitorContinueKind {
     before(node.label, tokens: .open)
-    after(node.colon, tokens: .break(.continue, size: 1))
+    after(node.colon, tokens: .break(.continue, newlines: .elective(ignoresDiscretionary: true)))
```

Comment:
This break had a `size` of 1 before, to create a space. Do we still want that space when it doesn't fire?

Reply:
The default size for a break is 1 (https://github.com/apple/swift-format/blob/master/Sources/SwiftFormatPrettyPrint/Token.swift#L210), so I was cleaning up the unnecessary specification while editing here.

We probably ought to do that throughout at some point. I also think we ought to make the break kind be explicit instead of defaulting to `.continue`.